src/vsr/replica.zig

```diff
@@ -6550,13 +6587,6 @@ const DVCQuorum = struct {
                 log_views_all.appendAssumeCapacity(log_view);
             }
         }
-
-        // Verify that DVCs with the same log_view do not conflict.
-        for (log_views_all.constSlice()) |log_view| {
-            const view_dvcs = dvcs_with_log_view(dvc_quorum, log_view);
-            var view_headers = HeaderIterator.init(view_dvcs, null);
-            while (view_headers.next()) |_| {}
-        }
```

Comment:
The loop above is dead code as well?

Reply:
Ah, I removed that intending to reimplement that check (since it can't use `HeaderIterator` anymore) but forgot to come back to it. I'll update that now.